setup.cfg

```diff
@@ -41,3 +48,8 @@ sa =
   sqlalchemy>=1.0,<1.4
 rsa =
   PyMySQL[rsa]>=1.0
+
+[options.packages.find]
+exclude =
+  tests
+  tests.*
```

Comment:
Are you sure both entries are needed?

Reply:
this was migrated from the current implementation, didn't review whether it's needed yet

Reply:
SQLAlchemy tests are in a separate `tests.sa` package, so both are indeed needed.